model_analyzer/config/input/config_command_profile.py

```diff
@@ -57,6 +57,7 @@
 from .objects.config_model_profile_spec import ConfigModelProfileSpec
 from .objects.config_protobuf_utils import \
     is_protobuf_type_primitive, protobuf_to_config_type
+from model_analyzer.result.model_constraints import ModelConstraints
```

Comment:
This import shouldn't be needed as ModelConstraints is not used in this file

Reply:
Removed import.